core/src/main/java/com/microsoft/applicationinsights/internal/config/TelemetryConfigurationFactory.java

```diff
@@ -85,7 +83,7 @@ public final void initialize(TelemetryConfiguration configuration) {
                 return;
             }
 
-            ApplicationInsightsXmlConfiguration  applicationInsightsConfig = builder.build(configurationFile);
+            ApplicationInsightsXmlConfiguration applicationInsightsConfig = builder.build(configurationFile);
```

Comment:
I think all this logic should be placed in a dedicated class, which is easier to test